seqr/views/apis/anvil_workspace_api.py

```diff
@@ -1,48 +1,112 @@
 """APIs for management of projects related to AnVIL workspaces."""
 
 import logging
-from django.contrib.auth.decorators import login_required
+import json
 
+from django.shortcuts import redirect
+from django.contrib.sites.shortcuts import get_current_site
+
+from seqr.models import Project, CAN_EDIT
+from seqr.views.utils.json_to_orm_utils import create_model_from_json
 from seqr.views.utils.json_utils import create_json_response
-from settings import API_LOGIN_REQUIRED_URL
+from seqr.views.utils.file_utils import load_uploaded_file
+from seqr.views.utils.terra_api_utils import add_service_account
+from seqr.views.utils.pedigree_info_utils import parse_pedigree_table
+from seqr.views.apis.individual_api import add_or_update_individuals_and_families
```

Comment:
you should never import between api files, as I mentioned before you need to move the helper method into the individual_utils file